nilearn/plotting/img_plotting.py

```diff
@@ -30,12 +31,32 @@
                                                  cbar_outline_set_xy)
 from .._utils.ndimage import get_border_data
 from ..datasets import load_mni152_template
+from ..image import iter_img
 from .displays import get_slicer, get_projector
 from . import cm
 
 ################################################################################
 # Core, usage-agnostic functions
 
+def black_to(color):
```

Comment:
Small remark, I think that @GaelVaroquaux wanted colormaps going from transparent to color and not from black to color.